test_conformance/spirv_new/CMakeLists.txt

```diff
@@ -0,0 +1,69 @@
+######################################################################################################
```

Comment:
this cmake is not a backport, it was written pretty much from scratch due to incompatibility between cl21 cmake and cl12 cmake